esp/bindings/http/platform/httpservice.cpp

```diff
@@ -400,35 +486,38 @@ int CEspHttpServer::processRequest()
                 if (!methodName.length())
                     return 0;
 
-                if (methodName.charAt(methodName.length()-1)=='_')
-                    methodName.setCharAt(methodName.length()-1, 0);
-                if (!stricmp(methodName.str(), "files"))
-                {
-                    if (!getTxSummaryResourceReq())
-                        ctx->cancelTxSummary();
-                    checkInitEclIdeResponse(m_request, m_response);
-                    return onGetFile(m_request.get(), m_response.get(), pathEx.str());
-                }
-                else if (!stricmp(methodName.str(), "xslt"))
+                if (untracedRequestIt != getRequests.end())
```

Comment:
picky style/encapsulation:  Rather than comparing against the map null entry I would add a None entry to UntracedGetMethod, and have a local variable of that type, which was filled in.  Why?  Because the later test is reasonably separated from where the variable is set up, and a test against getRequests.end() isn't immediately obvious that what it is checking.